inc/sendmail.php

```diff
@@ -7,6 +7,9 @@
 
 if ($_SERVER['REQUEST_METHOD'] == 'POST'){
 
+    //this is added here because if it added in index.php Swal.fire() Uncaught ReferenceError: Swal is not defined
+    echo "<script src='assets/js/sweetalert2.all.min.js'></script>";
```

Comment:
I think if you add this to the head section of index.php, it shouldn't give you the error.